express/blocks/app-store-highlight/app-store-highlight.css

```diff
@@ -0,0 +1,85 @@
+main .app-store-highlight-container {
+  padding: 40px 0;
+  background-color: var(--color-black);
+  color: var(--color-white);
+}
+
+main .app-store-highlight-container .content-wrapper {
+  transform: scale(1.2);
+}
+
+main .app-store-highlight-container .heading {
+  font-size: 36px;
+  line-height: 46px;
+  margin-bottom: 8px;
+}
+
+main .app-store-highlight-container .sub-heading {
+  font-size: 12px;
+  line-height: 18px;
+  margin: 0 0 16px;
+}
+
+main .app-store-highlight-container .rating-wrapper {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 8px;
+}
+
+main .app-store-highlight-container .rating-wrapper .icon-editor-choice {
+  width: 115px;
+}
+
+main .app-store-highlight-container .rating-stars {
+  font-size: 12px;
+  line-height: 18px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+main .app-store-highlight-container .rating-stars svg {
+  color: var(--color-premium);
+}
+
+main .app-store-highlight-container .rating-stars .rating-votes {
+  margin-left: 4px;
+}
+
+main .app-store-highlight-container .cards {
+  /*scroll-behavior: smooth;*/
+  display: flex;
+  gap: 12px;
+  padding: 53px 32px;
+  overflow: auto;
+  margin: 0 -32px;
+  -ms-overflow-style: none;  /* IE and Edge */
```

Comment:
We don't support IE, and isn't Egde `chromium` these days? I think this can go...